src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -17,6 +17,7 @@ import os from 'os';
 /**
  * Internal dependencies
  */
+import * as exit from 'lib/cli/exit';
```

Comment:
Out of curiosity, will this work?

```js
import exit from 'lib/cli/exit';
```

Reply:
Nope didn't work (it is still kinda mystery how these different JS module types work to me :)).